osquery/tables/system/darwin/xattr.cpp

```diff
@@ -0,0 +1,94 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#include <vector>
+#include <string>
+
+#include <sys/xattr.h>
+
+#include <glog/logging.h>
+
+#include <osquery/core.h>
+#include <osquery/tables.h>
+#include <osquery/filesystem.h>
+
+namespace osquery {
+namespace tables {
+
+std::string handleError(){
```

Comment:
Instead of returning these in results would it make more sense to `VLOG(1) << handleError()`?